ingestion_tools/dataset_configs/template.yaml

```diff
@@ -1,137 +1,188 @@
+      list_glob: REQUIRED, STRING (REGEX)
+      match_regex: OPTIONAL, STRING (REGEX) (DEFAULT '.*')
+      name_regex: OPTIONAL, STRING (REGEX) (DEFAULT '(.*)')
+    parent_filters: OPTIONAL
+      exclude: OPTIONAL
+        run: # can only exclude / include parent types (see IMPORTER_DEP_TREE)
+          - OPTIONAL, STRING (REGEX)
+      include: OPTIONAL
+        run: REQUIRED
+          - OPTIONAL, STRING (REGEX)
+  - source_multi_glob:
+      list_globs:
+        - REQUIRED, STRING
+    parent_filters: SEE source_glob.parent_filters
+  - destination_glob: OPTIONAL
+      list_glob: REQUIRED, STRING (REGEX)
+      match_regex: OPTIONAL, STRING (REGEX)
+      name_regex: OPTIONAL, STRING (REGEX)
+    parent_filters: SEE source_glob.parent_filters
+  - literal: OPTIONAL
+      value:
+      - REQUIRED, STRING
+    parent_filters: SEE source_glob.parent_filters
+frames: OPTIONAL
+- sources: SEE datasets.sources (source_glob, source_multi_glob)
+gains: OPTIONAL
+- sources: SEE datasets.sources (source_glob)
+key_images: OPTIONAL
+- sources: SEE dataset.sources (souce_glob, literal)
+rawtilts: OPTIONAL
+- sources: SEE datasets.sources (source_multi_glob)
+runs:
+- sources: SEE datasets.sources (source_glob
+standardization_config: REQUIRED
+  deposition_id: REQUIRED, INTEGER
+  run_data_map_file: OPTIONAL, STRING
+  run_to_frame_map_csv: OPTIONAL, STRING
+  run_to_tomo_map_csv: OPTIONAL, STRING
+  run_to_ts_map_csv: OPTIONAL, STRING
+  source_prefix: REQUIRED, STRING
+tiltseries: OPTIONAL
+- metadata: REQUIRED
+    acceleration_voltage: REQUIRED, INTEGER
+    aligned_tiltseries_binning: OPTIONAL, INTEGER (Default 1)
+    alignment_binning_factor: # DEPRECATED
+    binning_from_frames: OPTIONAL, FLOAT
+    camera: REQUIRED
+      manufacturer: REQUIRED, STRING
+      model: REQUIRED, STRING
+    data_acquisition_software: REQUIRED, STRING
+    is_aligned: REQUIRED, BOOLEAN (DEFAULT FALSE)
+    microscope: REQUIRED
+      additional_scope_info: OPTIONAL, STRING
+      manufacturer: REQUIRED, STRING
+      model: REQUIRED, STRING
+    microscope_optical_setup: REQUIRED
+      energy_filter: REQUIRED, STRING
+      image_corrector: OPTIONAL, STRING
+      phase_plate: OPTIONAL, STRING
+    pixel_spacing: REQUIRED, FLOAT
+    related_empiar_entry: OPTIONAL, STRING
+    spherical_aberration_constant: REQUIRED, FLOAT
+    tilt_alignment_software: OPTIONAL, STRING
+    tilt_axis: REQUIRED, FLOAT
+    tilt_range: REQUIRED
+      max: REQUIRED, FLOAT
+      min: REQUIRED, FLOAT
+    tilt_series_path: OPTIONAL, STRING # CURRENTLY NOT USED
+    tilt_series_quality: REQUIRED, INTEGER (1-5)
+    tilt_step: REQUIRED, FLOAT
+    tilting_scheme: REQUIRED, STRING
+    total_flux: REQUIRED, FLOAT
+  sources: REQUIRED
+  - source_glob: SEE dataset.sources.source_glob
+tomograms: OPTIONAL
+- metadata: REQUIRED
+    affine_transformation_matrix: OPTIONAL 4x4 FLOAT MATRIX (DEFAULT 4x4 identity matrix)
+    - - 1
```

Comment:
This one field feels out of place with an example, maybe we can add an example_config.yaml for example values in a future time?

Reply:
Sounds good! Added a TODO and removed the example.